<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ApplicationStartDlg</class>
 <widget class="QDialog" name="ApplicationStartDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Word Master</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#ApplicationStartDlg {
background: gray;
}

#mainFrame {
border: 3px solid gray;
border-radius: 40px;
background: QLinearGradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #eef, stop: 1 #ccf);
}

QPushButton {
color: white;
background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #88d, stop: 0.1 #99e, stop: 0.49 #77c, stop: 0.5 #66b, stop: 1 #77c);
border-width: 1px;
border-color: #339;
border-style: solid;
border-radius: 7;
padding: 3px;
font-size: 20px;
padding-left: 3px;
padding-right: 30px;
min-width: 150px;
max-width: 300px;
min-height: 13px;
max-height: 26px;
}</string>
  </property>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>370</y>
     <width>185</width>
     <height>34</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QFrame" name="mainFrame">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>431</width>
     <height>321</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>75</y>
      <width>258</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="wordFlashButton">
       <property name="text">
        <string>Word Flash</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="synonymMatchButton">
       <property name="text">
        <string>Synonym Match</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="antonymMatchButton">
       <property name="text">
        <string>Antonym Match</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="sentenceFillButton">
       <property name="text">
        <string>Sentence Fill</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="readingComprehensionButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>17</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Reading Comprehension</string>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>ApplicationStartDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>465</x>
     <y>324</y>
    </hint>
    <hint type="destinationlabel">
     <x>464</x>
     <y>257</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
